app/bundles/PageBundle/EventListener/ReportSubscriber.php

```diff
@@ -255,137 +255,20 @@ public function onReportBuilder(ReportBuilderEvent $event)
-                        'label' => 'mautic.page.report.hits.redirect_hit_count',
-                        'type'  => 'int',
-                    ],
-                    $redirectHit.'unique_hits' => [
-                        'label' => 'mautic.page.report.hits.redirect_unique_hits',
-                        'type'  => 'string',
-                    ],
-                    'ds.device' => [
-                        'label' => 'mautic.lead.device',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_brand' => [
-                        'label' => 'mautic.lead.device_brand',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_model' => [
-                        'label' => 'mautic.lead.device_model',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_os_name' => [
-                        'label' => 'mautic.lead.device_os_name',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_os_shortname' => [
-                        'label' => 'mautic.lead.device_os_shortname',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_os_version' => [
-                        'label' => 'mautic.lead.device_os_version',
-                        'type'  => 'string',
-                    ],
-                    'ds.device_os_platform' => [
-                        'label' => 'mautic.lead.device_os_platform',
-                        'type'  => 'string',
-                    ],
-                ];
-                $data = [
-                    'display_name' => 'mautic.page.hits',
-                    'columns'      => array_merge($columns, $hitColumns, $event->getLeadColumns(), $event->getIpColumn()),
-                ];
-                $event->addTable('page.hits', $data, 'pages');
-
-                // Register graphs
-                $context = 'page.hits';
-                $event->addGraph($context, 'line', 'mautic.page.graph.line.hits');
-                $event->addGraph($context, 'line', 'mautic.page.graph.line.time.on.site');
-                $event->addGraph($context, 'pie', 'mautic.page.graph.pie.time.on.site', ['translate' => false]);
-                $event->addGraph($context, 'pie', 'mautic.page.graph.pie.new.vs.returning');
-                $event->addGraph($context, 'pie', 'mautic.page.graph.pie.devices');
-                $event->addGraph($context, 'pie', 'mautic.page.graph.pie.languages', ['translate' => false]);
-                $event->addGraph($context, 'table', 'mautic.page.table.referrers');
-                $event->addGraph($context, 'table', 'mautic.page.table.most.visited');
-                $event->addGraph($context, 'table', 'mautic.page.table.most.visited.unique');
-            }
+
+            $event->addTable(self::CONTEXT_PAGE_HITS, $data, self::CONTEXT_PAGES);
```

Comment:
These changes comes from my PR - I suppose you just merged staging into this PR? Will be better to rebase this PR to see changes.

Reply:
Hm, maybe not - this seems like changes? `pages` was replaced by page_hits...